test/passes/minify-imports-and-exports-and-modules.txt

```diff
@@ -0,0 +1,12 @@
+longname1 => a
+longname2 => b
+longname3 => c
+(module
+ (type $none_=>_none (func))
+ (import "a" "memory" (memory $0 256 256))
+ (import "a" "table" (table $0 4 funcref))
+ (import "a" "a" (func $internal1))
+ (import "a" "b" (func $internal2))
+ (import "a" "c" (func $internal3))
+ (import "a" "longname4" (func $internal4))
```

Comment:
Interesting that we can smoosh everything from "env" and "other" into the "a", but can't touch longname4. Presumably this is because these are user-provided symbols?

Reply:
Oh, good point, after thinking on this some more I think this is inconsistent. The idea was that we want to minify the normal things like `env.` and `wasi.`, and we leave alone stuff we don't recognize. However, if the user asked to minify modules, then we are definitely not leaving anything along - we are even changing module basenames. So we should minify all the things on those modules. Updating for that.